classes/03class/exercises/c03-compose01/danielvca/README.md

```diff
@@ -0,0 +1,29 @@
+﻿
+# C03-COMPOSE01
+
+## Make
+- [docker-compose.yaml](docker-compose.yaml)
+
+## Command Execution Output
+
+- Execution and output of the command `docker-compose ps`
+
+```
+CONTAINER ID IMAGE COMMAND CREATED STATUS PORTS NAMES  
```

Comment:
It looks like the output of `docker ps` and not `docker-compose ps`.

Both serve a similar purpose with one difference: `docker ps` will list ALL running containers and `docker-compose ps` will show only those one from the docker-compose at your current project (folder with docker-compose.yaml).

Could you please update with the output of `docker-compose ps`? 🙂